### PR TITLE
Automate Lottie exporter tests using Travis

### PR DESCRIPTION
**Pull request by AnishGG**
Resolves: #989 
- The job is currently allowed to fail because there are around 6 tests that are failing.

One doubt:
- Where should I place the script `lottie_test.sh`? Currently, it is inside the root directory, but I suppose this is not the right place for it.

### CODE REVIEW

File: lottie_test.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Don't abort on Error
+# set -e
+
+DIRECTORY=`pwd`
```
Comment (ice0) [inline lottie_test.sh]: It is better to use the same approach as in the build script:

https://github.com/synfig/synfig/blob/4f6e09c576477df12dc2726efbcaee25f1914288/2-build-debug.sh#L28

File: lottie_test.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        ((Error_count=Error_count+1))
+        ((pass=0))
+    }
+
+    # To display whether a file ran successfully or not
+    if [ $pass -eq 1 ]
+    then
+        echo "[OK] $filename";
+    else
+        echo "[FAIL] $filename";
```
Comment (ice0) [inline lottie_test.sh]: I suggest colorizing the output (this will be better for UX)

**Comment by AnishGG** [comment]
@ice0 have made the suggested changes. Could you review this ^__^

**Comment by AnishGG** [comment]
@ice0 I have made the required changes. Please review ^_^

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by morevnaproject** [comment]
@AnishGG Thanks a lot for this!

**Pull request merged**